### PR TITLE
atool: Add package

### PR DESCRIPTION
**Pull request by nikil**
Maintainer: me
Compile tested: ar71xx, x86_64
Run tested: x86_64

Description: tool for managing file archives of various types

Signed-off-by: Nikil Mehta nikil.mehta@gmail.com

### CODE REVIEW

File: utils/atool/Makefile [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=atool
+PKG_VERSION:=0.39.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://savannah.nongnu.org/download/atool/
```
Comment (diizzyy) [inline utils/atool/Makefile]: PKG_SOURCE_URL:=@SAVANNAH/$PKG_NAME

Use the available URL variables (which also provides mirrors)

**Comment by diizzyy** [comment]
This may sound stupid but why is need at all when you have libarchive (bsdtar, bsdcpio) available?
https://github.com/openwrt/packages/blob/master/libs/libarchive/Makefile
Just submit a pull req instead with changed options, it's a much better choice in the end...

File: utils/atool/Makefile [hunk 2]
```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
```
Comment (diizzyy) [inline utils/atool/Makefile]: Please don't use OpenWrt.org copyright unless you are under contract.
https://www.lede-project.org/faq.html#who_holds_the_copyright_of_lede_code

File: utils/atool/Makefile [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=atool
+PKG_VERSION:=0.39.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://savannah.nongnu.org/download/atool/
+PKG_MD5SUM:=e0aa006decbc6484b9dba2879f1bb9f0
```
Comment (diizzyy) [inline utils/atool/Makefile]: Please use SHA256 hashes instead of MD5

File: utils/atool/Makefile (hunk 1 above)
Comment (nikil) [inline utils/atool/Makefile]: Oh nice, I haven't seen that before.  Thanks.

**Comment by nikil** [comment]
No worries, that's not a stupid question.  The main advantage of atool is that it has some smart batch processing that not in bsdtar.  With arepack you can convert all archives of one type to another with a single command.  bsdtar is nice but I find this package pretty useful.

I'm a new contributor to openwrt and am unsure how packages are accepted, so I'm happy with whatever you decide.

**Comment by diizzyy** [comment]
Hmm... Given the rather limited usefulness and due to the fact that libarchive (and others) can accomplish this even if it a bit more work I don't think this needs to be in the main repo.

@hnyman @thess 
Comments?

**Comment by thess** [comment]
@nikil - submissions may be ignored because the existing maintainers and reviewers have little or no interest in the package. As far as this package goes, I have no objections other than it is ancient, un-maintained, has limited usefulness and requires Perl. I vote with @diizzyy,

**Comment by diizzyy** [comment]
@thess 
Close as not being accepted in the end?

**Comment by thess** [comment]
Old PR being closed for lack of interest.

**Pull request closed without merging**